doc/cprover-manual/unsound_options.md

```diff
@@ -0,0 +1,37 @@
+# Unsound CLI options for various tools
+
+Be advised that the following command line options to `cbmc` and `goto-instrument`
+have been reported to be unsound:
+
+* `--full-slice` has been reported to be unsound in issue [cbmc#260](https://github.com/diffblue/cbmc/issues/260)
+  In particular, `--full-slice` appears to lead to spurious counter examples,
+  because values that get assigned by a function whose body gets sliced out are
+  no longer present in the trace, but still result in flipped verification results.
+* `--reachability-slice` has been reported unsound at least once, in issue [cbmc#6394](https://github.com/diffblue/cbmc/issues/6394).
+  The state of this option these days is that this flag *should not* contain significant
+  soundness bugs, but it has a number of potential issues that pop up everyone now and then,
+  which renders it unsuitable for production use.
```

Comment:
I think we might want to list other options in here, but I'm not sure `--reachability-slice` still warrants this warning. On the other hand, `--aggressive-slice` would fit in here.